### PR TITLE
New site structure

### PR DESCRIPTION
**Pull request by acev0010**
## What does this pull request (PR) do? / Que fait cette demande « pull » (PR)?
Add tags and creates sub-landing pages under /main/ folder for:

- about-us
- accessibility-fundamentals
- accessibility-in-the-government
- how-tos
- resources-and-tools

(Both English and French files)

## Additional information (optional) / Information additionnelle (optionel)

Running npm: start gives the following error: 

```
[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] 1. Having trouble rendering njk template ./src/main/en/about-us/contact-us.njk (via TemplateContentRenderError)
[11ty] 2. (./src/main/en/about-us/contact-us.njk)
[11ty]   TypeError: Cannot read properties of undefined (reading 'filter') (via Template render error)
```

This error happens with all the files I modified, I am unsure why it happens. 

I will leave this pull request open until I resolve this problem.

### CODE REVIEW

**Comment by acev0010** [comment]
Update: 

npm:start works now but the website isn't working properly.

When you go to a page, for example `about us > contact us` it doesn't take the user to the contact us page. Instead, it generates an extra page that loops the same page over and over again:

image

I will come back to this next week

**Comment by shawnthompson** [comment]
That would mean on the Contact Us page you are using `tags: contactUs`

**Review by shawnthompson — CHANGES_REQUESTED** [review]

File: src/main/en/about-us/about-the-digital-the-accessibility-toolkit-project.njk [hunk 1]
```diff
@@ -2,7 +2,9 @@
 title: About the Digital the accessibility toolkit project
 description: Learn more about this project and what the Digital Accessibility Toolkit is.
 layout: layouts/base.njk
-tags: aboutUs
+tags:
+  - aboutUs
+  - aboutTheDigitalTheAccessibilityToolkitProject
```
[the remark this answers is not in the document]
Comment (acev0010) [reply src/main/en/about-us/about-the-digital-the-accessibility-toolkit-project.njk]: I removed the tags but ''contact us'' still loops around, is that normal?

**Comment by acev0010** [comment]
The following pages were either missing a sub-category or didn't have tags from the table:

**en:**

- mobility-flexibility-and-body-structure-disabilities.md
- universal-design-for-learning-udl.md

**fr:**

- conception-universelle-pour-lapprentissage-cua.md
- handicaps-cognitifs.md
- handicaps-de-mobilite-de-flexibilite-et-de-structure-corporelle.md
- mandats.md
- project-de-la-boite-a-outils-de-laccessibilite-numerique

**Review by shawnthompson — APPROVED** [review]

**Pull request merged**